### PR TITLE
Fix `deleted_item_key` in InventoryItem

### PR DESCRIPTION
**Pull request by laverdet**
This will fix GetInventoryItems while releasing pokemon. There may be
other things in this payload, but I haven't seen them yet.

### CODE REVIEW

**Comment by laverdet** [comment]
See: #111

File: src/POGOProtos/Inventory/InventoryItem.proto [hunk 1]
```diff
@@ -5,6 +5,10 @@ import "POGOProtos/Inventory/InventoryItemData.proto";
 
 message InventoryItem {
 	int64 modified_timestamp_ms = 1;
-	int64 deleted_item_key = 2;
+	DeletedItemKey deleted_item_key = 2;
 	.POGOProtos.Inventory.InventoryItemData inventory_item_data = 3;
+
+	message DeletedItemKey {
```
Comment (rubenvereecken) [inline src/POGOProtos/Inventory/InventoryItem.proto]: Can we please just name it DeletedItem, both the message name and the key name.

**Comment by laverdet** [comment]
Updated

**Comment by rubenvereecken** [comment]
Nice one!

**Pull request merged**